api/src/test/java/edu/wpi/first/shuffleboard/api/sources/CompositeNetworkTableSourceTest.java

```diff
@@ -1,18 +1,18 @@
-package edu.wpi.first.shuffleboard.sources;
+package edu.wpi.first.shuffleboard.api.sources;
```

Comment:
`CompositeNetworkTableSource` should be in `:app`, so this should be moved as well